tokio-trace-tower/src/lib.rs

```diff
@@ -27,8 +27,8 @@ where
 {
     type Request = T::Request;
     type Response = T::Response;
-    type Future = Instrumented<T::Future>;
     type Error = T::Error;
+    type Future = Instrumented<T::Future>;
```

Comment:
> This change seems unrelated, can we back it out?

Right, sorry.

> I'm assuming this is due to us running differing versions of rustfmt --- are you on nightly?

 No, this was IntelliJ's lint saying that the implementation was out of order with the defintion. Do you want me to revert?

Reply:
> Do you want me to revert?

As long as it still passes `rustfmt`, I don't _really_ care. In general I try to keep unrelated formatting changes out of commits that add or modify code so that the git history is clearer, but this is so minor it doesn't really matter to me.